testing/matrix-sdk-integration-testing/src/tests/sliding_sync/room.rs

```diff
@@ -538,3 +551,354 @@ async fn test_room_notification_count() -> Result<()> {
 
     Ok(())
 }
+
+/// Boolean that decides if to_device messages should be dropped.
+static DROP_TODEVICE: StdMutex<bool> = StdMutex::new(true);
```

Comment:
Instead of global state, could we store this bool in the `CustomResponder` struct?